.github/workflows/build.yaml

```diff
@@ -29,9 +29,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v2.4.0
```

Comment:
We can use v3.0. Is there any reason not to use v3.0?
https://github.com/actions/checkout#checkout-v3

Reply:
Great catch, I missed that one. :pray: Fixing.

Reply:
All set! Thank yous. :smile:

Reply:
Thanks for your work!